### PR TITLE
[tools/shoestring]: allow collect of multiple certificates date

### PR DESCRIPTION
**Pull request by Wayonb**
Update to use ``openssl storeutl`` to parse all certificates instead of ``openssl x509`` which only returns information for the first.
``openssl storeutl`` will fail if the input is not a valid pem, extract the certificate information from the output of ``openssl s_client``.

### CODE REVIEW

File: tools/shoestring/shoestring/healthagents/rest_https_certificate.py
Comment (Jaguar0625) [inline tools/shoestring/shoestring/healthagents/rest_https_certificate.py:42]: use f string

File: tools/shoestring/shoestring/healthagents/rest_https_certificate.py
Comment (Jaguar0625) [inline tools/shoestring/shoestring/healthagents/rest_https_certificate.py:39]: add back blank line for readability

File: tools/shoestring/shoestring/healthagents/rest_https_certificate.py
Comment (Jaguar0625) [inline tools/shoestring/shoestring/healthagents/rest_https_certificate.py:31]: "cert" and "certificate" are being used interchangeably in variable names, consider picking one and using it throughout

File: tools/shoestring/shoestring/healthagents/rest_https_certificate.py
Comment (Jaguar0625) [inline tools/shoestring/shoestring/healthagents/rest_https_certificate.py:84]: why do you need this condition vs just returning true at end if len(collected_dates) != 1

Comment (Wayonb) [reply tools/shoestring/shoestring/healthagents/rest_https_certificate.py:84]: Currently, it verifies that the certificates found by OpenSSL match the number of dates collected.
If this is a best-effort scenario, then it can check that at least one date is collected.

**Review by Jaguar0625 — APPROVED** [review]
looks good to me but wait for @gimre-xymcity review before merging

File: tools/shoestring/shoestring/healthagents/rest_https_certificate.py
Comment (gimre-xymcity) [inline tools/shoestring/shoestring/healthagents/rest_https_certificate.py:58]: this probably should not require `command input` ?

**Pull request merged**
